Because we are not using USEARCH, this patch will set deafult search engine to used BLAST+
Author: Filip Kružík (INUITS)

--- DAS_Tool.orig	2022-01-18 16:13:32.781571000 +0100
+++ DAS_Tool	2022-01-18 16:15:43.039420000 +0100
@@ -52,7 +52,7 @@
     echo "   -o, --outputbasename       Basename of output files."
     echo "   -l, --labels               Comma separated list of binning prediction names. (optional)"
     echo "   --search_engine            Engine used for single copy gene identification [blast/diamond/usearch]."
-    echo "                              (default: usearch)"
+    echo "                              (default: blast)"
     echo "   --write_bin_evals          Write evaluation for each input bin set [0/1]. (default: 1)"
     echo "   --create_plots             Create binning performance plots [0/1]. (default: 1)"
     echo "   --write_bins               Export bins as fasta files  [0/1]. (default: 0)"
@@ -117,7 +117,7 @@
 create_plots=1
 write_bins=0
 write_unbinned=0
-search_engine="usearch"
+search_engine="blast"
 b_weight=0.6
 c_weight=0.5
 db_dir=$DIR\/db
